modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordWriterRegistryImpl.java

```diff
@@ -63,14 +62,8 @@ protected void addDDMFormInstanceRecordWriter(
 		String type = MapUtil.getString(
 			properties, "ddm.form.instance.record.writer.type");
 
-		String extension = MapUtil.getString(
-			properties, "ddm.form.instance.record.writer.extension");
-
-		if (Validator.isNull(extension)) {
-			extension = StringUtil.toUpperCase(type);
-		}
-
-		_ddmFormInstanceRecordWriterExtensions.put(type, extension);
+		_ddmFormInstanceRecordWriterExtensions.put(
+			type, StringUtil.toUpperCase(type));
```

Comment:
@dantewang @JanisZhang Please check with the from team why this property is not used and if they still need this, this was added by https://issues.liferay.com/browse/LPS-81565. 

Before the property is added, we do not have this map. So, if we want to remove, we should also remove this map _ddmFormInstanceRecordWriterExtensions.

This way, we do not need the extra logic in service tracker.

Reply:
@tinatian Janis has asked in Slack, and the product team has agreed that we can get rid of the "extension" property. So @JanisZhang as per Tina's comment, let's also remove this map, and replace the code that uses the "extension" with "type".